strings/word_replacement.json

```diff
@@ -64,3 +64,78 @@
         "wine": "vino"
     }
 }
+
+
+    "french": {
+		"yes": "oui"
+		"no": "non"
+        "I'm": "j'", 
+        "am": "un", 
```

Comment:
suis. un means "a" (masculine)

Reply:
Duly noted; I only speak English, German, and Japanese, so I figured there'd be at least a few errors in the grammar department, as the stereotypical cartoon french accent is a lot more focused around articles, pronouns and similar